### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by BackpackXl**
I will try to translate the port of android melonds in Spanish

### CODE REVIEW

**Comment by rafaelvcaetano** [comment]
Thanks for your amazing contribution! I see that there are still some strings left to translate, so take your time. Do let me know when you think you have finished!

**Comment by BackpackXl** [comment]
There is a way to test the translation

**Comment by rafaelvcaetano** [comment]
For now, the only way to test is it by making a local build on your machine. Then, you just need to set your device's language to Spanish and the app will appear in Spanish. You can find instructions on how to make a local build in the project's README.

But let me know if you don't know how to do it or have some issue. I may be able to arrange something! But I can already tell you that the app will crash when opening the General settings. To fix it, you will need to update your branch and remove the `theme_options` string array from your file.

**Comment by BackpackXl** [comment]
Thank. I will try

**Comment by BackpackXl** [comment]
translations are finished are fine

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+        <item>Micrófono del dispositivo</item>
+    </string-array>
+
+    <string-array name="console_type_options">
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="fps_counter_position_options">
+        <item>escondido</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:333]: Capitalization is wrong.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+        <item>Arriba-izquierda</item>
+        <item>Centro-superior</item>
+        <item>Arriba-derecha</item>
+        <item>Abajo-izquierda</item>
+        <item>inferior-central</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:338]: Capitalization is wrong.

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+        <item>Abajo-derecha</item>
+    </string-array>
+
+    <string-array name="video_filtering_options">
+        <item>ninguno</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:343]: Capitalization is wrong, as well as in the string below.

File: app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+        <item>lineal</item>
+        <item>2xBR</item>
+        <item>HQ2X</item>
+        <item>HQ4X</item>
+        <item>Quilez</item>
+        <item>LCD</item>
+    </string-array>
+
+    <string-array name="audio_interpolation_options">
+        <item>ninguno</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:353]: All strings in this array have wrong capitalization.

File: app/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+    </string-array>
+
+    <string-array name="audio_latency_options">
+        <item>Bajo (más lento)</item>
+        <item>Medio</item>
+        <item>Alto (más rápido)</item>
+    </string-array>
+
+    <string-array name="mic_source_options">
+        <item>ninguno</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:372]: Wrong capitalization in this string and the one below.

File: app/src/main/res/values-es/strings.xml [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+
+    <string-array name="save_state_locations">
+        <item>Guardar directorio de archivos</item>
+        <item>Directorio de ROMs</item>
+        <item>Directorio interno</item>
+    </string-array>
+
+    <string-array name="layout_orientation_options">
+        <item>Seguir sistema</item>
+        <item>retrato</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:385]: Wrong capitalization in this string and the one below.

File: app/src/main/res/values-es/strings.xml [hunk 7]
```diff
@@ -0,0 +1,402 @@
...
+        <item>paisaje</item>
+    </string-array>
+
+    <string-array name="background_portrait_mode_options">
+        <item>elasticidad</item>
+        <item>Centro de ajuste</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:391]: This translation has a name but the original text has a verb. What do you think about using "Ajuste al centro" instead? The same could apply to the same string in the `background_landscape_mode_options` array below.

File: app/src/main/res/values-es/strings.xml [hunk 8]
```diff
@@ -0,0 +1,402 @@
...
+    </string-array>
+
+    <string-array name="layout_orientation_options">
+        <item>Seguir sistema</item>
+        <item>retrato</item>
+        <item>paisaje</item>
+    </string-array>
+
+    <string-array name="background_portrait_mode_options">
+        <item>elasticidad</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:390]: Wrong capitalization, as well as in the same string in the array below.

File: app/src/main/res/values-es/strings.xml [hunk 9]
```diff
@@ -0,0 +1,402 @@
...
+    <string name="hint_search_roms">Buscando ROMs…</string>
+    <string name="update_available">Actualización disponible: %1$s</string>
+    <string name="skip_update">Omitir</string>
+    <string name="downloading_update">Descargando actualización…</string>
+    <string name="starting_download">Iniciando descarga…</string>
+    <string name="download_progress_sizes">%1$.1fMB/%2$.1fMB</string>
+    <string name="update_download_failed">Error al descargar la actualización</string>
+
+    <string name="dsiware_manager">Administrador de DSiWare</string>
+    <string name="no_dsiware_titles_installed">no tienes ningún título DSiWare instalado</string>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:55]: The capitalization in this string is incorrect, as well as in the string below.

File: app/src/main/res/values-es/strings.xml [hunk 10]
```diff
@@ -0,0 +1,402 @@
...
+        <item>Centro de ajuste</item>
+        <item>Ajuste superior</item>
+        <item>Ajuste inferior</item>
+    </string-array>
+
+    <string-array name="background_landscape_mode_options">
+        <item>elasticidad</item>
+        <item>Centro de ajuste</item>
+        <item>Ajuste a la izquierda</item>
+        <item>Ajuste correcto</item>
```
Comment (rafaelvcaetano) [inline app/src/main/res/values-es/strings.xml:400]: The proper translation here is likely to be "Ajuste a la derecha".

**Review by rafaelvcaetano — CHANGES_REQUESTED** [review]
I have some issues in your translations and one of them simple prevents the app from compiling. Do you think you could fix them?

Additionally, I have made some translation suggestions. I am not a native spanish speaker, but I deal with the spanish language every day so I like to think I know a thing or two :P Feel free to completely throw away my suggestions because they can be completely stupid.

File: app/src/main/res/values-es/strings.xml [hunk 11]
```diff
@@ -0,0 +1,402 @@
...
+    <string name="starting_download">Iniciando descarga…</string>
+    <string name="download_progress_sizes">%1$.1fMB/%2$.1fMB</string>
+    <string name="update_download_failed">Error al descargar la actualización</string>
+
+    <string name="dsiware_manager">Administrador de DSiWare</string>
+    <string name="no_dsiware_titles_installed">no tienes ningún título DSiWare instalado</string>
+    <string name="dsiware_manager_no_dsi_setup">no tienes configurado el sistema DSi</string>
+    <string name="dsiware_manager_invalid_dsi_setup">La configuración del sistema DSi no es válida</string>
+    <string name="import_dsiware_title">Importar título de DSiWare</string>
+    <string name="dsiware_manager_setup">configuración</string>
```
[the remark this answers is not in the document]
Comment (BackpackXl) [reply app/src/main/res/values-es/strings.xml]: I made more mistakes in forgetting to capitalize

[the remark this answers is not in the document]
Comment (BackpackXl) [reply app/src/main/res/values-es/strings.xml]: It's solved

File: app/src/main/res/values-es/strings.xml [hunk 12]
```diff
@@ -0,0 +1,402 @@
...
+    <string name="category_roms_summary">Directorio de búsqueda, filtros de iconos, configuración de caché</string>
+    <string name="category_system">Sistema</string>
+    <string name="category_system_summary">Configuración de firmware, BIOS customizado, JIT</string>
+    <string name="theme">Tema</string>
+    <string name="fast_forward_max_speed">Velocidad máxima avance rápido</string>
+    <string name="rewind_description">Cuando está habilitado, un estado de guardado se guarda periódicamente de forma automática. Siempre que desee retroceder en su progreso por un corto período de tiempo, simplemente abra la pantalla de rebobinado y seleccione el estado desde el que desea reanudar.\n\nTenga en cuenta que al habilitar esta función, puede experimentar tartamudeos ocasionales si su dispositivo no es lo suficientemente potente. También se utiliza una cantidad considerable de memoria dependiendo de la frecuencia con la que desee que se capture el estado y durante cuánto tiempo se debe mantener.</string>
+    <string name="rewind_save_period">Período de guardado</string>
+    <string name="rewind_length">duración de rebobinado</string>
```
[the remark this answers is not in the document]
Comment (BackpackXl) [reply app/src/main/res/values-es/strings.xml]: It's solved

File: app/src/main/res/values-es/strings.xml [hunk 13]
```diff
@@ -0,0 +1,402 @@
...
+    <string name="dsi_bios_directory">Directorio de DSi BIOS</string>
+    <string name="show_boot_screen">Mostrar pantalla de arranque</string>
+    <string name="mac_address">Dirección MAC</string>
+    <string name="randomize_mac_address">Aleatorizar la dirección MAC</string>
+    <string name="randomize_mac_address_summary">Cuando está habilitado, se usa una dirección MAC aleatoria en cada arranque. Algunos juegos pensarán que se están ejecutando en un sistema diferente y puede romper eventos basados en el tiempo.</string>
+    <string name="generate_new_mac_address">Generar nueva</string>
+    <string name="generate_new_mac_address_description">Generar una nueva dirección MAC hará que los juegos piensen que se están ejecutando en un sistema diferente.</string>
+    <string name="enable_jit">Habilitar JIT</string>
+    <string name="enable_jit_summary">Mejora el rendimiento, pero puede provocar bloqueos o crasheos</string>
+    <string name="jit_not_supported">JIT compatible con dispositivos de 64 bits.</string>
```
[the remark this answers is not in the document]
Comment (BackpackXl) [reply app/src/main/res/values-es/strings.xml]: It's true it looks better

File: app/src/main/res/values-es/strings.xml [hunk 14]
```diff
@@ -0,0 +1,402 @@
...
+    <string name="threaded_rendering">Renderizado de subprocesos</string>
```
[the remark this answers is not in the document]
Comment (BackpackXl) [reply app/src/main/res/values-es/strings.xml]: makes more sense

**Comment by BackpackXl** [comment]
I have already solved everything. If there are more errors I will not hesitate to fix it

**Comment by rafaelvcaetano** [comment]
Everything looks good! Thanks for your amazing contribution!

**Review by rafaelvcaetano — APPROVED** [review]

**Pull request merged**